src/assets/styles/components/_chat.scss

```diff
@@ -0,0 +1,393 @@
+    textarea {
+      height: 32px;
+    }
+    .v-text-field {
+      padding-top: 0;
+      margin-top: 0;
+    }
+    .v-text-field__slot {
+      font-weight: 400;
+    }
+  }
+  &__form-append {
+    margin-right: 8px;
+  }
+  &__fab {
+    position: absolute;
+    right: 16px;
+    bottom: 16px;
+    opacity: 0.7;
+    &:hover {
+      opacity: 1;
+    }
+  }
+}
+
+/** Themes **/
+.theme--light {
+  .a-chat {
+    &__content {
+      color: map-get($shades, 'black');
+    }
+    &__message {
+      background-color: #fff;
+      box-shadow: 0 1px 10px hsla(0, 0%, 39.2%, 0.06),
+        0 1px 1px hsla(0, 0%, 39.2%, 0.04),
+        0 2px 10px -1px hsla(0, 0%, 39.2%, 0.02);
+      &--red {
+        background-color: map-get($red, 'lighten-5');
+      }
+      &--green {
+        background-color: map-get($green, 'lighten-5');
+      }
+    }
+    &__sender {
+      color: map-get($grey, 'darken-4');
+    }
+    &__timestamp {
+      color: map-get($adm-colors, 'muted');
+    }
+    &__blockchain-status {
+      color: map-get($adm-colors, 'muted');
+    }
+    &__amount {
+      color: map-get($adm-colors, 'regular');
+    }
+    &__rates {
+      color: map-get($adm-colors, 'muted');
+    }
+    &__direction {
+      color: map-get($adm-colors, 'regular');
+    }
+    &__message-text {
+      color: map-get($adm-colors, 'regular');
+
+      a {
+        color: map-get($blue, 'base');
+      }
+      pre {
+        background-color: map-get($grey, 'lighten-4');
+        color: map-get($grey, 'darken-3');
+        border-color: map-get($grey, 'lighten-2');
+      }
+    }
+    &__form {
+      background-color: map-get($adm-colors, 'secondary2');
+      .v-text-field__slot {
+        .v-label {
+          color: map-get($adm-colors, 'muted');
+        }
+        textarea {
+          color: map-get($adm-colors, 'regular');
+          caret-color: map-get($adm-colors, 'primary2');
+        }
+      }
+      .v-text-field > .v-input__control > .v-input__slot:before {
+        border-color: rgba(0, 0, 0, 0.12);
+      }
+      .v-text-field:not(.v-input--has-state)
+        > .v-input__control
+        > .v-input__slot:hover:before {
+        border-color: rgba(0, 0, 0, 0.12);
+      }
+      .v-text-field.v-input--is-focused
+        > .v-input__control
+        > .v-input__slot:after {
+        border-width: thin 0 0 0;
+        color: map-get($adm-colors, 'primary2');
+      }
+      .v-text-field > .v-input__control > .v-input__slot:after {
+        transition: unset;
+      }
+      .v-input--is-focused .v-input__slot .v-icon {
+        color: map-get($adm-colors, 'muted') !important;
+      }
+      &--is-active .v-input--is-focused .v-input__slot .v-icon {
+        color: map-get($adm-colors, 'primary') !important;
+      }
+    }
+  }
+}
+.theme--dark {
+  .a-chat {
+    &__content {
+      color: map-get($shades, 'white');
+    }
+    &__message {
+      background-color: #000;
+      box-shadow: 0 1px 10px hsla(0, 0%, 39.2%, 0.06),
```

Comment:
create variable for `box-shadow` value

Reply:
also the value of this property is the same as in `theme--light`, so it must be defined on 76(?) line

Reply:
no refactoring in this MR